tools/VscPreviewExtension/src/Processors/previewProcessor.ts

```diff
@@ -38,13 +41,30 @@ export class PreviewProcessor {
 
     private updateContentCoreAsync(uri: Uri) {
         let that = this;
-        PreviewProcessor.proxy.newRequest(uri, PreviewProcessor.previewType, PreviewProcessor._context, function (err,response) {
+        PreviewProcessor.proxy.newRequest(this.prepareRequestData(uri, PreviewProcessor.previewType, PreviewProcessor.context, function (err, response) {
             if (err) {
                 window.showErrorMessage(`[Proxy Error]: ${err}`);
             } else {
                 that.pageRefresh(response);
             }
-        });
+        }));
+    }
+
+    private prepareRequestData(documentUri: Uri, previewType: number, context, callback): ProxyRequest {
+        let environmentVariables;
+        try {
+            environmentVariables = Utility.getEnvironmentVariables();
+        } catch (err) {
```

Comment:
>catch (err) [](start = 10, length = 11)

if any exceptions may be thrown in getEnv method, should this logic put into getEnv method? initializeTempFileInformation also calls getEnv method as well